README.md

```diff
@@ -1,3 +1,86 @@
 # xdggs
 
-Xarray extension for DGGS
+`xdggs` is an extension for [Xarray](https://xarray.pydata.org/) that provides tools to handle geospatial data using Discrete Global Grid Systems (DGGS). It allows efficient manipulation and analysis of multi-dimensional gridded data within a DGGS framework, facilitating spatial data processing, resampling, and aggregations on both global and regional scales.
+
+## Key Features
+
+- **Seamless Integration with Xarray**: Use `xdggs` alongside Xarray's powerful tools for managing labeled, multi-dimensional data.
+- **Support for DGGS**: Convert geospatial data into DGGS representations, allowing for uniform spatial partitioning of the Earth's surface.
+- **Spatial Resampling**: Resample data on DGGS grids, enabling downscaling or upscaling across multiple resolutions.
+- **DGGS Aggregation**: Perform spatial aggregation of data on DGGS cells.
+- **Efficient Data Management**: Manage large datasets with Xarray's lazy loading, Dask integration, and chunking to optimize performance.
+
+## Installation
+
+To install `xdggs`, you can clone the repository and install it using pip:
+
+```bash
+git clone https://github.com/xarray-contrib/xdggs.git
+cd xdggs
+pip install .
+```
+
+Alternatively, you can install it directly via pip (once it's available on PyPI):
+
+```bash
+pip install xdggs
+```
+
+## Demo
+
+![xdggs demo](xdggs-cropped.gif)
+
+## Getting Started
+
+As an example, this is how you would use `xdggs` to reconstruct geographical coordinates from the cell ids then make interactive plotting indicating cell ids and associated Latitude and Longitude coordinate:
+
+```python
+import xarray as xr
+import xdggs
+
+# Load your Xarray dataset made using ./examples/prepare_dataset_h3.ipynb
```

Comment:
```suggestion
# Load the dataset created by ./examples/prepare_dataset_h3.ipynb
```
Though in the future we should definitely create a set of tutorial datasets.

Reply:
Will you change and merge ? or I need to change?